### PR TITLE
Spelling check and correction

### PR DESCRIPTION
**Pull request by vb138**
I performed a complete check and correction of some errors

### CODE REVIEW

File: resources/translations/cs_CZ/cooling/cool_min_layer_time.md [hunk 1]
```diff
@@ -10,4 +10,4 @@ Toto nastavení má tři efekty:
 * Pokud se vrstva tiskne rychleji, než je uvedeno v parametru [Práh 
  normální/maximální rychlosti ventilátoru](cool_min_layer_time_fan_speed_max.md), zvýší se rychlost ventilátoru tak, aby dosáhl parametru [Maximální rychlost ventilátoru](cool_fan_speed_max.md). Pokud je vrstva tak malá, že její tisk trvá kratší dobu, použije se maximální rychlost ventilátoru. Rychlost ventilátoru je mezi nimi interpolována.
 * Pokud vytisknutí vrstvy zabere méně času, než je minimální doba vrstvy, rychlost tisku se zpomalí, takže vždy zabere minimální čas vrstvy.
-* Pokud je tisková hlava příliš pomalá (tisk pomalejší než nastavení parametru [Minimální rychlost](cool_min_speed.md)), bude tisková hlava čekat na konci vrstvy a může ji trochu zvednout.
+* Pokud je tisková hlava příliš pomalá (tisk pomalejší než nastavení parametru [Minimální rychlost](cool_min_speed.md), bude tisková hlava čekat na konci vrstvy a může ji trochu zvednout.
```
Comment (Ghostkeeper) [inline resources/translations/cs_CZ/cooling/cool_min_layer_time.md:13]: Oh this one was correct I think. One closing bracket is from the link, the other one from the actual bracketed text that started at the word `tisk`.

File: resources/translations/cs_CZ/cooling/cool_min_speed.md [hunk 2]
```diff
@@ -2,7 +2,7 @@ Minimální rychlost
...
+Je důležité respektovat minimální dobu vrstvy, aby předchozí vrstva mohla ztuhnout, než položí na vrch další vrstvu. Snížení rychlosti tisku je užitečné, protože ventilátory tiskové hlavy mohou na vlákno déle foukat. Na druhé straně je nadměrné snížení rychlosti tisku škodlivé, protože teplo trysky se může šířit do spodních vrstev a na sousední stěny. To může způsobit, že povrch bude velmi špinavý a způsobí místní ochabnutí. Proto existuje také minimální rychlost. Na konci vrstvy se tisková hlava normálně pohybuje trochu nahoru (pokud není deaktivován parametr [Zvednout hlavu](cool_lift_head.md), který umožňuje ventilátoru foukat na vlákno o něco déle, dokud neuplyne minimální doba vrstvy. Tiskárna poté přejde na další vrstvu.
```
Comment (Ghostkeeper) [inline resources/translations/cs_CZ/cooling/cool_min_speed.md:5]: Similarly, you removed a closing bracket here that I think you didn't intend to remove. The bracketed text starts at the word `pokud`.

File: resources/translations/cs_CZ/troubleshooting/unfinished_print.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ Poruchy tiskárny
 ----
 Příčinou neúplného tisku je někdy selhání tiskárny, když tiskárna nedělá to, co se od ní očekává. Obecně platí, že Cura může udělat jen málo pro nápravu těchto problémů. Zde jsou některé běžné příčiny.
 * Výpadek napájení. Cura s tím nemůže dělat nic jiného než rychlejší tisk, aby se snížilo riziko výpadku napájení během tisku.
-* Porucha napájení. Pokud je jedna z teplot (buď [podložky tisku](../material/material_bed_temperature.md) nebo [trysky](../material/material_print_temperature.md)), příliš vysoká, některé tiskárny mají poruchový režim, kde napájecí zdroj shoří nebo teplotní senzory vydávají chyby. Tiskárna se poté vypne a nedokončí tisk.
```
Comment (Ghostkeeper) [inline resources/translations/cs_CZ/troubleshooting/unfinished_print.md:11]: This is removing a closing bracket, so the brackets don't match any more. The bracketed text starts with the word `buď`.

File: resources/translations/cs_CZ/cooling/cool_min_layer_time.md (hunk 1 above)
Comment (vb138) [reply resources/translations/cs_CZ/cooling/cool_min_layer_time.md]: fixed

File: resources/translations/cs_CZ/cooling/cool_min_speed.md (hunk 2 above)
Comment (vb138) [reply resources/translations/cs_CZ/cooling/cool_min_speed.md]: fixed

File: resources/translations/cs_CZ/troubleshooting/unfinished_print.md (hunk 3 above)
Comment (vb138) [reply resources/translations/cs_CZ/troubleshooting/unfinished_print.md:11]: fixed

**Pull request merged**